package.json

```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "chalk": "^2.4.2",
+    "express": "^4.17.0",
```

Comment:
What about lodash, is this not needed by the cli for the same reason?

Reply:
Here, `express` is moved to dependencies because it is imported within the `cli` folder. `lodash` is not used there, so I guess it is fine? I guess another move that should be done is to remove `express` from `peerDependencies`?

Reply:
oh right 👍